spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -6308,16 +6350,17 @@ disable transaction management for a test or for the whole class as follows:
 ----
 
 
-A list of the auto-configuration that is enabled by `@DataNeo4jTest` can be
+A list of the auto-configuration settings that are enabled by `@DataNeo4jTest` can be
 <<appendix-test-auto-configuration#test-auto-configuration,found in the appendix>>.
 
 
 
 [[boot-features-testing-spring-boot-applications-testing-autoconfigured-redis-test]]
-==== Auto-configured Data Redis tests
-`@DataRedisTest` can be used if you want to test Redis applications. By default, it will
-scan for `@RedisHash` classes and configure Spring Data Redis repositories. Regular
-`@Component` beans will not be loaded into the `ApplicationContext`:
+==== Auto-configured Data Redis Tests
+You can use`@DataRedisTest` to test Redis applications. By default, it
```

Comment:
Missing space between use and `@DataRedisTest`

Reply:
Yup. Thanks.